src/Tasks/MigrateContentToElement.php

```diff
@@ -33,6 +34,9 @@ public function run($request)
                 $element->HTML = $content;
                 $element->ParentID = $area->ID;
                 $element->write();
+                if (class_exists(Versioned::class)) {
```

Comment:
```suggestion
                if ($page->hasExtension(Versioned::class)) {
```

Versioned is a dependency so it will always exist 🙂

Reply:
Versioned will always been an extension actually so this condition can just be removed.